lib/engine-addon.js

```diff
@@ -66,7 +65,7 @@ function buildConfigTree(prefix) {
   // Include a module that reads the engine's configuration from its
   // meta tag and exports its contents.
   var configContents = this.getEngineConfigContents();
-  var configTree = writeFile(prefix + '/config/environment.js', configContents);
```

Comment:
Yep, that's the gist of it. Gotta find a way to make these changes play nicely with non-master ember-cli though